### PR TITLE
Confirm deletion on the wishlists#show view

### LINKED ISSUE #101: Display confirmation dialog before deletion on the wishlist#show view

### PR DESCRIPTION
**Pull request by student-tomasz**
Closes #101 

screenshot 2016-07-28 13 33 46

### CODE REVIEW

**Comment by agalkiewicz** [comment]
:+1:

File: app/views/wishlists/index.html.erb [hunk 1]
```diff
@@ -36,14 +36,14 @@
 
               <%= link_to t("edit"), edit_wishlist_path(wishlist), class: "btn btn-sm btn-default" -%>
 
-              <a class="btn btn-sm btn-danger-outline btn-primary" data-toggle="popover" data-placement="left"
-                  data-trigger="focus" tabindex="0"
-                  data-content-source="#wlist-<%= wishlist.id -%>-delete">
+              <a class="btn btn-sm btn-danger-outline btn-primary" data-toggle="popover-with-html"
+                  data-placement="left" data-trigger="focus" tabindex="0"
```
Comment (agalkiewicz) [inline app/views/wishlists/index.html.erb]: indentation

Comment (student-tomasz) [inline app/views/wishlists/index.html.erb]: Well... I think it's correct to indent the unclosed tag's attributes one step further, to show it's still the same scope, but in a new line. Then a next tag on the normal plus-one-step indent.

Comment (agalkiewicz) [inline app/views/wishlists/index.html.erb]: Ok then, sorry if I anger you. I'm not too familiar with conventions on html indentations.

Comment (student-tomasz) [inline app/views/wishlists/index.html.erb]: Nah, I'm not angry at all. It wasn't my intent to sound angry at all. I just write my comments straight to the point, so I guess I tend to skip some pleasantries along the way.

**Comment by Kacper3331** [comment]
👍

File: app/views/wishlists/show.html.erb [hunk 2]
```diff
@@ -18,9 +18,15 @@
       <span class="glyphicon glyphicon-edit"></span> Edit list description
     <% end %>
 
-    <%= link_to @wishlist, method: :delete, class: "btn btn-default" do %>
+    <a class="btn btn-sm btn-danger-outline btn-primary" data-toggle="popover-with-html"
+        data-placement="top" data-trigger="focus" tabindex="0"
+        data-content-source=".wlist-<%= @wishlist.id -%>-delete-confirm">
```
Comment (gotar) [inline app/views/wishlists/show.html.erb]: extra space (wrong indentation)

File: app/views/wishlists/index.html.erb (hunk 1 above)
Comment (gotar) [inline app/views/wishlists/index.html.erb]: it should be align (indent) at the same level as element above (so no extra space) just the same level as class

**Pull request merged**